fw/ss_skb.c

```diff
@@ -393,6 +393,7 @@ __split_linear_data(struct sk_buff *skb_head, struct sk_buff *skb, char *pspt,
 	}
 	skb->tail -= tail_len;
 	skb->data_len += tail_len;
+	skb->truesize += tail_len;
```

Comment:
Please look for this place carefully! This code is necessary to prevent warning in kernel with small MTU and it makes akb truesize correct. But i don't fully understand why

Reply:
As I see it's correct fix. We need to adjust `truesize` when we move data from linear part to paged fragments.

Lat's see on `truesize` in kernel, it's consist of `size of skb data + sizeof(sk_buff) + sizeof(skb_shared_info)`. It means size of reserved linear data will be included in `truesize` during skb allocating. Because we don't change preallocated size of skb head when we manipulate with it, we must not change `truesize`.
However, when we move data from head to paged we increase size of skb(`data_len`) by adding new fragment, even if it points to already allocated data within skb. Thus, we must increment `truesize`. Just for clarity, if we will realloc the skb, we can allocate less data to headroom and in this case we can not change `truesize`.

Reply:
Little addition: We must not update `truesize` when we delete linear data from skb. Now looks like we do that.